src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx

```diff
@@ -29,34 +31,34 @@ const Title = styled(Text)`
 `;
 
 export interface IAccordionHeader {
-  /** determines the position of the chevron icon */
-  isOpen: boolean;
+  id: string;
+  index: number;
   textSize: ITextProps['size'];
```

Comment:
It looks like `textSize` is defaulting to `"medium"` so there's no need to make it required?
 ( _I know it was like this already_ 🙈  )